nextflow/configs/profiles/sumner2.config

```diff
@@ -388,13 +432,68 @@ process {
     withLabel: "r_clip_video" {
         cpus = 2
         // 0.0000003 * t_min + 0.35
-        memory = { ((0.0000003 * params.clip_duration / 30 / 60 + 0.35) * 1.5).toInteger() + '.GB' * task.attempt }
+        memory = {
+            def r_value = Math.max(1, ((0.0000003 * (params.clip_duration / 30 / 60) + 0.35) * 2.5 * task.attempt).toInteger())
+            return r_value + '.GB'
+        }
         // 19.8 * t_min + 52
-        time = { ((19.8 * params.clip_duration / 30 / 60 + 52) * 2.5).toInteger() + '.sec' * task.attempt }
+        time = {
+            def r_value = Math.max(1, ((52 + 19.8 * (params.clip_duration / 30 / 60)) * 2.5 * task.attempt).toInteger())
+            return r_value + '.sec'
+        }
+        array = 200
+        errorStrategy = 'retry'
+        maxRetries = 3
+    }
+    // Multi Mouse Resources
+    // These processes assume the input video is 1hr and scale with number of animals
+    withLabel: "r_multi_seg" {
+        cpus = 2
+        // 8.53 GB * num_mice
+        memory = {
+            def r_value = Math.max(1, ((8.53 * params.num_mice) * 2.5 * task.attempt).toInteger())
+            return r_value + '.GB'
+        }
+        time = { 5.hours * task.attempt }
+        array = 200
+        errorStrategy = 'retry'
+        maxRetries = 3
+    }
+    withLabel: "r_multi_keypoints" {
+        cpus = 2
+        // 2.26 GB * num_mice + 2.47 GB
+        memory = {
+            def r_value = Math.max(1, ((2.26 * params.num_mice + 2.47) * 2.5 * task.attempt).toInteger())
+            return r_value + '.GB'
+        }
+        // 6 * num_mice + 3
+        time = {
+            def r_value = Math.max(1, ((3 + 6 * params.num_mice) * 2.5 * task.attempt).toInteger())
+            return r_value + '.hours'
+        }
+        array = 200
+        errorStrategy = 'retry'
+        maxRetries = 3
+    }
+    withLabel: "r_multi_identity" {
+        cpus = 2
+        memory = { 2.GB * task.attempt }
+        // 0.1 * num_mice + 0.16
+        time = {
+            def r_value = Math.max(1, ((0.1 * params.num_mice + 0.16) * 2.5 * task.attempt).toInteger())
+            return r_value + '.hours'
+        }
         array = 200
         errorStrategy = 'retry'
         maxRetries = 3
     }
+    withLabel: "r_multi_tracklets" {
+        cpus = 1
+        // TODO: Tune these numbers based on number of mice
+        // First video suggested ~7GB and ~30min for 3 mice
+        memory = { 10.GB * task.attempt }
```

Comment:
I believe this should start at `16.GB`, but I also think it doesn't need to double with every attempt.